build.gradle

```diff
@@ -208,7 +209,11 @@ dependencies {
     provided "appeng:RotaryCraft:${config.rotc.version}:api"
     provided ("appeng:appliedenergistics2:${config.ae2.version}:dev") {
         exclude module: 'buildcraft'
-    }
+    }*/
+
```

Comment:
No not the whole thing, just the bits that use things not in the API as they may change. I know the old code uses the whole thing but it's generally not a good idea. And I do recall the reason this code is not already back in OC is because Alpha mods are not supported